tests/wdio.sauce.conf.js

```diff
@@ -0,0 +1,89 @@
+const { config } = require('./wdio.default.conf')
+
+exports.config = {
+  ...config,
+
+  user: process.env.SAUCE_USERNAME,
+  key: process.env.SAUCE_ACCESS_KEY,
+
+  maxInstances: 5,
+  capabilities: [
+    {
+      browserName: 'firefox',
+      platform: 'Windows 10',
+    },
+    {
+      browserName: 'firefox',
+      platform: 'macOS 10.14',
+    },
+    {
+      browserName: 'chrome',
+      platform: 'Windows 10',
+    },
+    {
+      browserName: 'chrome',
+      platform: 'macOS 10.14',
+    },
+    {
+      browserName: 'MicrosoftEdge',
+      platform: 'Windows 10',
+    },
+    {
+      browserName: 'internet explorer',
+      platform: 'Windows 8.1',
+      version: '11',
+    },
+    {
+      browserName: 'internet explorer',
+      platform: 'Windows 8',
+      version: '10',
+    },
+    {
+      browserName: 'internet explorer',
+      platform: 'Windows 7',
+      version: '9',
+    },
+    /* note the change from platform to platformName
+     * that goes with the selenium version switch
+     */
+    {
+      browserName: 'safari',
+      browserVersion: 'latest',
+      platformName: 'macOS 10.14',
+      'sauce:options': { seleniumVersion: '3.14.0' },
+    },
+    /* this fails on a click handler in helpers.js
```

Comment:
Let's also make sure to cover this too & evaluate if it's indicative of something we should look into.